next/src/lib/orm/data-source.ts

```diff
@@ -0,0 +1,40 @@
+import * as dotenv from 'dotenv';
+import * as path from 'path';
+import { DataSource, ObjectLiteral, ObjectType, Repository } from 'typeorm';
+
+import { DataCheck } from './entity/DataCheck';
+import { Record } from './entity/Record';
+import { User } from './entity/User';
+
+dotenv.config({
+  path: path.resolve(__dirname, '../../../.env.local'),
+});
```

Comment:
```suggestion
```


I would prefer to not rely on a file with env variables, but rather on just setting environmental variables. Can we remove this, and instead have this in docker compose:

```
env_file: ./next/.env.local
```

I tested locally and seems to work just fine.